runtime-modules/content/src/tests/nft/accept_incoming_offer.rs

```diff
@@ -182,43 +183,49 @@ fn accept_incoming_offer_no_incoming_offers() {
 
 #[test]
 fn accept_incoming_offer_with_nft_owner_being_a_member_channel() {
+    let video_id = 1u64;
     with_default_mock_builder(|| {
         // Run to block one to see emitted events
         run_to_block(1);
-
-        let video_id = NextVideoId::<Test>::get();
-        create_initial_storage_buckets_helper();
-        increase_account_balance_helper(DEFAULT_MEMBER_ACCOUNT_ID, INITIAL_BALANCE);
-        create_default_member_owned_channel_with_video();
-
-        UpdateChannelFixture::default()
+        // channel with no reward account
+        CreateChannelFixture::default()
+            .with_sender(DEFAULT_MEMBER_ACCOUNT_ID)
+            .with_actor(ContentActor::Member(DEFAULT_MEMBER_ID))
+            .call();
+        CreateVideoFixture::default()
             .with_sender(DEFAULT_MEMBER_ACCOUNT_ID)
             .with_actor(ContentActor::Member(DEFAULT_MEMBER_ID))
             .call_and_assert(Ok(()));
 
-        // Issue nft
         assert_ok!(Content::issue_nft(
             Origin::signed(DEFAULT_MEMBER_ACCOUNT_ID),
             ContentActor::Member(DEFAULT_MEMBER_ID),
             video_id,
             NftIssuanceParameters::<Test>::default(),
         ));
```

Comment:
Now I remembered why I refactored most of the setups to use `ContentTest`.
It was because storage buckets need to be initialized before channel is created + in general it's just a lot of boilerplate to create a channel, video and nft. I suggest using:
```suggestion
        ContentTest::with_member_channel().with_video_nft().setup();
```